buildspec.yml

```diff
@@ -0,0 +1,18 @@
+version: 0.2
+
+phases:
+  pre_build:
+    commands:
+      - yarn
+  build:
+    commands:
+      - yarn build-storybook
```

Comment:
we need additional step to build icons -> `yarn build`. Alternatively we can run this command from `yarn build-storybook`. Building icons for storybook is something we need to fix anyway

Reply:
No worries, I will add `yarn build` before that step. Thanks for catch 😄

Reply:
Done